python/lsst/sims/utils/timeTransformations.py

```diff
@@ -0,0 +1,195 @@
+"""
+For definitions of all (or, at least, most of) the time systems handled in this file, see
+
+https://www.cv.nrao.edu/~rfisher/Ephemerides/times.html
+"""
+
+import numpy as np
+import warnings
+import palpy
+from lsst.sims.utils import Ut1MinusUtcData
+from lsst.sims.utils import TT_from_TAI_Data
+
+__all__ = ["taiFromUtc", "utcFromTai",
+           "dutFromUtc",
+           "ut1FromUtc", "utcFromUt1",
+           "dttFromUtc", "ttFromTai",
+           "tdbFromTt"]
+
+
+def taiFromUtc(utc):
+    """
+    Use the palpy method palDat to convert from Coordinated Universal Time
+    (UTC) to International Atomic Time (TAI)
+
+    @param [in] utc is the UTC time as an MJD
+
+    @param [out] TAI time as an MJD
+    """
+
+    dt = palpy.dat(utc) # returns TAI-UTC in seconds
+    return utc + dt/86400.0
+
+
+def utcFromTai(tai):
+    """
+    Use the palpy method palDat to convert from International Atomic Time
+    (TAI) to Coordinated Universal Time (UTC)
+
+    @param [in] tai is the TAI time as an MJD
+
+    @param [out] utc time as an MJD
+    """
+
+    # because the PALPY method only returns TAI-UTC as a function
+    # of UTC, we will have to construct an array of TAI and UTC
+    # values using taiFromUtc and interpolate along them
+
```

Comment:
See Tim's comment below :) Pal (& palpy) is not intended to cover all these extra functions, which seems like a reasonable stance since it's just supposed to replace SLALIB.